x-pack/plugins/cases/public/components/user_actions/common.tsx

```diff
@@ -51,10 +51,8 @@ export const createCommonUpdateUserActionBuilder = ({
   label,
   icon,
   handleOutlineComment,
-}: BuilderArgs): ReturnType<UserActionBuilder> => ({
-  // TODO: Fix this manually. Issue #123375
-  // eslint-disable-next-line react/display-name
-  build: () => [
+}: BuilderArgs): ReturnType<UserActionBuilder> => {
```

Comment:
note: I changed this component to remove the eslint ignore. 

The way this is fixed is by creating the function inside a variable, assigning the `displayName` property and then returning it instead of inlining the function in the returned object.

Reply:
The `CommonUpdateUserActionBuilder` is not a React component but an object with a property (`username`) that contains an object. By doing that you create an object of the form:

```
{
   build: () => { ... }
   displayName: 'CommonUpdateUserActionBuilder'
}
````

For some reason, eslint and TS (you should get an error about the extra property as it is not part of the signature) are tricked.

Reply:
@cnasikas it is using jsx inside therefore it is recognized as a react component. I rather leave it like this, it doesn't affect the functionality and remove the eslint-ignore. What do you think?

Reply:
As it is not a React component I think is better to leave it as it is. The eslint is not smart enough to recognize it.